Project-II-Calculator/calculator/src/components/ButtonComponents/Button.css

```diff
@@ -0,0 +1,42 @@
+.action{
+    color: #FFFFFF;
+    background-color: #9F001E;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex: 1;
+    font-size: 2.3rem;
+    font-weight: 200;
+    width: 58px;
+    border: 0;
+    border-bottom: 1px solid #7A7A7A;
+  }
+  
+   .clear, .zero {
+    width: 170px;
+    border-top: 0;
+    border-left: 0;
+    font-size: 1.4rem;
+    border-color: #7A7A7A;
+  }
+  
+   .clear {
+    font-weight: 200;
+  }
+  
+   .zero {
+    font-size: 1.3rem;
+    font-weight: 700;
+  }
+  
+   .number {
+    border-color: #7A7A7A;
+    display: flex;
+    justify-content: center;
```

Comment:
Note that display flex is for a parent container, not sure why you are adding to these